src/systems/joint_state_publisher/JointStatePublisher.cc

```diff
@@ -51,33 +51,68 @@ void JointStatePublisher::Configure(
     return;
   }
 
-  // Create the position, velocity, and force components for the joint.
-  std::vector<Entity> childJoints = _ecm.ChildrenByComponents(
-      this->model.Entity(), components::Joint());
-  for (const Entity &joint : childJoints)
+  // If a joint_name is specified in the plugin, then only publish the
+  // specified joints. Otherwise, publis all the joints.
```

Comment:
publis -> publish

Reply:
Fixed in: 36b7b3e5f83179be42e8b67d9cd2ecb71dd81046